src/bci_build/package/apache_tomcat.py

```diff
@@ -73,6 +76,8 @@ def _get_sac_supported_until(
         supported_until=_get_sac_supported_until(
             os_version=os_version, tomcat_ver=tomcat_ver, jre_major=jre_version
         ),
+        build_flavor=f"openjdk{jre_version}",
+        all_build_flavors=[f"openjdk{ver}" for ver in all_jre_versions],
```

Comment:
Taking my idea about one single item, I think something like this could be done here.

```python
        build_flavors=[Flavor(name=f"openjdk{ver}", current=ver==jre_version) for ver in all_jre_versions],
```

Reply:
I think this is way too complicated for a simple container definition, and would have to repeated for each one of them (lets think about the python containers becoming multibuild, for example).

Reply:
Yes, it was just an idea, because I wasn't that happy with the solution proposed.